warehouse/templates/accounts/profile.html

```diff
@@ -66,12 +66,12 @@ <h1 class="author-profile__name">{{ user.name }}</h1>
       <h2>
         {% if projects %}
           {% trans count=projects|length %}
-            {{ count }} project
+            {{ count }} active project
           {% pluralize %}
-            {{ count }} projects
+            {{ count }} active projects
           {% endtrans %}
         {% else %}
-          {% trans %}No projects{% endtrans %}
+          {% trans %}No active projects{% endtrans %}
```

Comment:
Not sure I would go as far as to say these are "active" (maybe the maintainer hasn't discovered the archive feature yet?) but I'm on the fence.

Reply:
Yeah, good point! I could just drop this and tweak the count so that it includes the archived projects, WDYT?